plutarch-test/plutarch-test.cabal

```diff
@@ -96,11 +96,18 @@ library
   hs-source-dirs:  src
   exposed-modules: Plutarch.Test
 
+  -- FIXME remove -Wwarn=deprecations
+  ghc-options:     -Wwarn=deprecations
```

Comment:
Should we have an issue for this perhaps? Using deprecated functions in examples that users might look at is probably not the best look.

Reply:
Maybe. wrt. the examples, they will all be ported to the new testing framework anyway, and wrt. plutarch-test, that will be fixed in another PR as it's a lot of work.

Reply:
is @srid working on this? or did you mean you're working on it in another PR? or does someone else need to work on it?

Reply:
@srid is working on this